deployment/helm/nri-plugins/templates/role.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: nri-resource-policy
+  namespace: {{ .Values.namespace }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - watch
+- apiGroups:
+  - topology.node.k8s.io
+  resources:
+  - noderesourcetopologies
```

Comment:
Hmm, wouldn't we actually need ClusterRole for managing cluster-scoped objects?

Reply:
Fixed in https://github.com/containers/nri-plugins/pull/59